services/wfm/src/main/java/org/openkilda/wfm/ctrl/ClearStateAction.java

```diff
@@ -0,0 +1,19 @@
+package org.openkilda.wfm.ctrl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.annotations.VisibleForTesting;
+import org.openkilda.wfm.MessageFormatException;
+import org.openkilda.wfm.UnsupportedActionException;
+
+@VisibleForTesting
```

Comment:
You'll see several comments on using @VisibleForTesting .. I think there are conventions within google guava for how they use it .. it is unclear how/why it is being used here. Most of the elements being tagged are public.

Whereas it is clear what "ClearState" should do .. and it may be useful for production scenarios in any case .. probably need a bit more documentation on this. AFAIK, this is included in the main code, not just test code. This brings up issues, let's discuss.

Reply:
When I write a test for that code I stuck with an initialization problem, Test load and init bolt only one for all use cases. So, I have no choice and create clear endpoint in bolt for simulate clean(non init) state of bolt without reload all storm topology. I used VisibleForTesting for that test code.If we will use that code for production I will remove that annotation and write some documentation.

Reply:
I think you have a couple of options: 
1) since this is just for testing ... does it make sense to recreate the topology for each test scenario (ie use @Before and @After, vs. @BeforeClass and @AfterClass
2) remove the "public" and just make it package scope, ie don't put public / private / protected. This way, the test scenario can still access it.

Reply:
1. I tried to do that, spend a lot of time and result was very unstable, storm & kafka didn't response and crash sometimes I tried to find what is wrong and spend another day.
2.  I will try to do that.